packages/traceability-schemas/schemas/IntentToImport.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$id": "https://w3id.org/traceability/schemas/IntentToImport.json",
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$comment": "{\"term\": \"IntentToImport\", \"@id\": \"https://w3id.org/traceability#IntentToImport\"}",
+    "title": "IntentToImport",
+    "description": "A simple notification by an entity (person or organization) notifying authorities of intention to import goods into its jurisdiction. Basic attributes are included only for the purpose of letting the authority determine the required documentation needed for the given import. This is a highly simplified and should not be interpreted as a replacement of Import Declarations or other relevant documents. This object may be easily subclassed for extension with other data for other cases where additional regulatory data may be required.",
+    "type": "object",
+    "properties": {
+        "@context": {
+            "type": "array"
+        },
+        "type": {
+            "oneOf": [
+                {
+                    "type": "string"
+                },
+                {
+                    "type": "array"
+                }
+            ]
+        },
+        "importer": {
```

Comment:
I may be missing something but doesn't this need to also reference a specific product? 
As far as I could tell all this is saying is "This organization intends to import", it does not reference where it is importing or what is it importing?

Reply:
[Intent To Sell](https://w3c-ccg.github.io/traceability-vocab/#IntentToSell) I think actually provides this capability with product definition (if available) - perhaps we rename that or merge these two objects - @OR13 thoughts?

Reply:
I think the idea is to put that in other credentials presented along with this one.

Reply:
Indeed, the idea was to have the ITI bundled with other credentials. I had an earlier PR with an ITI which included commodity code, but we decided to take that out. Which leads to the question: do we even need an ITI at all?